bool askDelay
---
float32 sendDelay
